src/preferences/dialog/dlgprefeffectsdlg.ui

```diff
@@ -464,9 +464,16 @@
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutMetaKnob">
       <item>
-       <widget class="QRadioButton" name="radioButtonKeepMetaknobPosition">
+       <widget class="QRadioButton" name="radioButtonKeepAllMetaknobPositions">
         <property name="text">
-         <string>Keep metaknob position</string>
+         <string>Keep all metaknob positions</string>
+        </property>
+       </widget>
+      </item>
+      <item>
+       <widget class="QRadioButton" name="radioButtonKeepTopMetaknobPosition">
+        <property name="text">
+         <string>Only keep the top-level Effect Unit metaknob position</string>
```

Comment:
```suggestion
         <string>Restore meta knob position when loading a chain preset and keep it when changing an effect</string>
```

Reply:
Below: "Always restore meta knob position"

Reply:
We may consider to ditch the middle option, because the user may be either used to changing meta knobs or to fixed. 
Making this conditional to the effect selector used might be surprising.